src/modm/driver/rtc/mcp7941x.hpp

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2021, Odin Holmes
+ * Copyright (c) 2021, Raphael Lehmann
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * chip datasheet available at http://ww1.microchip.com/downloads/en/DeviceDoc/22266A.pdf
```

Comment:
Ups.

Reply:
@rleh What is the issue? Just the formatting or anything else I am to blind to see right now?

Reply:
The datasheet link should go into the module.lb instead of the license header. THis particular formatting makes it look like it's part of the license as well.

Reply:
Unrelated, the `struct mcp7941x` should also have a `/// @ingroup modm_driver_mcp7941x` on top of it, otherwise doxypress will not group it correctly (sadly).

Reply:
Line 10 does not belong in the license header. I probably copy-pasted it in there by mistake.

Reply:
Of course, I was blind ... Will fix it as well.